src/biome/text/dataset_readers/mixins.py

```diff
@@ -133,6 +136,5 @@ def build_textfield(
                 self._token_indexers,
             )
             for value in data
-            if value
```

Comment:
Maybe i would leave this line as is to prevent `TextField`s with empty strings in the `ListField`. I will check my notes on how this would affect the further processing. Any specific reason why you would like to remove this line?

Reply:
We have ambiguity problems in this statement with numpy/pandas types (`numpy.arrays` or `pandas.Series` are not "evaluables" falling into the error:

````shell
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
````

Reply:
Generating the textual form of value before empty checks will avoid ambiguity of some types